modules/components/src/NestedTreeView/NestedTreeView.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactTreeView from './ReactTreeView';
+import './style.css';
+
+const Leaf = ({ title }) => (
+  <div className="tree-view_children leaf">{title}</div>
+);
+
+const NestedTreeView = ({ dataSource }) =>
+  dataSource.map(
+    (node, i) =>
+      node.children ? (
+        <ReactTreeView
+          key={i}
+          nodeLabel={node.title}
+          defaultCollapsed={true}
+          itemClassName={node.isHeader && 'header'}
+        >
+          {node.children ? <NestedTreeView dataSource={node.children} /> : null}
```

Comment:
if `node.children` is falsey this line will never be reached anyways from the check above

Reply:
Good point... :P